TKGame/BackEnd/GameDebug.cs

```diff
@@ -37,10 +38,8 @@ public static void UpdateFPS(GameTime gameTime)
         /// <summary>
         /// Draw bounding rectangle around a given rectangle.
         /// </summary>
-        public static void DrawBoundingBox(Rectangle rectangle, Color color, int lineWidth)
+        public static void DrawBoundingBox(Rectangle rectangle, Color color, int lineWidth, SpriteBatch spriteBatch)
```

Comment:
This was to test something else, This should be removed and utilize TKGame.SpriteBatch